frontend/src/component/common/ConstraintsList/ValuesList/ValuesList.tsx

```diff
@@ -0,0 +1,66 @@
+import type { FC } from 'react';
+import { styled, Tooltip } from '@mui/material';
```

Comment:
Most other places (that I know of) either use the tooltip resolver (`frontend/src/component/common/TooltipResolver/TooltipResolver.tsx`) or the HTML tooltip (`frontend/src/component/common/HtmlTooltip/HtmlTooltip.tsx`) directly. This should also make it look like it does in the sketches and match other tooltips we have.

Reply:
Will do in next PR